$MirOS: ports/plan9/kencc/patches/patch-utils_kl_l_h,v 1.1 2006/03/31 03:50:21 tg Exp $
--- utils/kl/l.h.orig	Thu Mar 30 23:54:36 2006
+++ utils/kl/l.h	Thu Mar 30 23:55:24 2006
@@ -282,7 +282,7 @@ void	dodata(void);
 void	doprof1(void);
 void	doprof2(void);
 long	entryvalue(void);
-void	errorexit(void);
+void	errorexit(void) __attribute__((__noreturn__));
 void	exchange(Prog*);
 int	find1(long, int);
 void	follow(void);
